src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -785,26 +786,28 @@ private void SetAffinity(Process process, int cores, LicenseLimits licenseLimits
 
         private bool ShouldIgnoreProcessorAffinityChanges(int cores, LicenseLimits licenseLimits)
         {
-            if (IgnoreProcessorAffinityChanges == false)
-                return false;
-
-            if (licenseLimits == null)
+            if (IgnoreProcessorAffinityChanges)
```

Comment:
Should ignore and then return false? Is this a case of double negation?

Reply:
`IgnoreProcessorAffinityChanges` is static and used only for the tests not to change the affinity of the process.
As far as I understand we always want to skip setting the affinity in the test.